aslam_cv_cameras/include/aslam/cameras/undistorter-mapped-inl.h

```diff
@@ -88,9 +87,16 @@ inline std::unique_ptr<MappedUndistorter> createMappedUndistorter(
   common::buildUndistortMap(
       *input_camera, *output_camera, CV_16SC2, map_u, map_v);
 
-  return std::unique_ptr<MappedUndistorter>(new MappedUndistorter(
-      input_camera, output_camera, map_u, map_v, interpolation_type));
+  // Convert map to non-fixed point representation for easy lookup of values.
+  cv::Mat map_u_float = map_u.clone();
+  cv::Mat map_v_float = map_v.clone();
+  aslam::convertMapsLegacy(map_u, map_v, map_u_float, map_v_float, CV_32FC1);
```

Comment:
why do we need to use a legacy function in this core functionality? Is this reverting the map convention to legacy?